package.json

```diff
@@ -9,7 +9,7 @@
   },
   "main": "dist/npm/index.js",
   "module": "dist/es/module.js",
-  "browser": "dist/streaming-client-browser.js",
+  "browser": "dist/streaming-client.browser.js",
```

Comment:
It may be worth it (for tracking purposes) to open a PCM ticket to tie this too. That way it will have a history you can tie the commit, version, and dev center issue to. Also, the CHANGELOG should be updated with this change too since it does affect consumption of sc.

Reply:
Updated.